src/index.html

```diff
@@ -11,7 +11,194 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST" onsubmit="onSubmit()">
+      <fieldset class="fieldset_distance fieldset__first">
+        <legend>
+          Personal information
+        </legend>
+
+        <div class="form-field input input__first">
+          <label for="surname">
+            Surname:
+          </label>
+
+          <input type="text" id="surname"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">
+            Name:
+          </label>
+
+          <input type="text" id="name"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="old">
+            How old are You?
+          </label>
+
+          <input type="number" value="12"
+            min="12" max="100"
+            id="old" required
+          >
```

Comment:
If there are more than two attributes - each should start on new line. Fix it for the whole document.
```suggestion
          <input
            type="number"
            value="12"
            min="12"
            max="100"
            id="old"
            required
          >
```